test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/CaseSensitiveTagHelperAttributeComparer.cs

```diff
@@ -22,16 +21,13 @@ public bool Equals(IReadOnlyTagHelperAttribute attributeX, IReadOnlyTagHelperAtt
                 attributeX == attributeY ||
                 // Normal comparer doesn't care about the Name case, in tests we do.
                 string.Equals(attributeX.Name, attributeY.Name, StringComparison.Ordinal) &&
-                Equals(attributeX.Value, attributeY.Value);
+                Equals(attributeX.Value, attributeY.Value) &&
+                attributeX.Minimized == attributeY.Minimized;
```

Comment:
There's no base class for this comparer. It's comparing the `IReadOnlyTagHelperAttribute`. This wasn't touched in your PR. Will update to do the `Minimized == true` then no value check though.